docs/aws-copilot.md

```diff
@@ -390,3 +390,30 @@ The following sites contain some valuable examples that can be used for inspirat
 
 - https://asecure.cloud/l/cloudwatch/
 - https://www.intelligentdiscovery.io/controls/cloudwatch/
+
+## Server Access Logging on S3 Buckets
+
+This is a feature provided by AWS. Server access logging for S3 buckets in AWS provides detailed records for the requests made to a specific bucket. These logs are invaluable for security and audit purposes.
+
+### Enabling Server Access Logging
+
+This can be done through the Management Console as well as the AWS CLI, but for the sake of convenience, we will only conver doing it though former.
```

Comment:
```suggestion
This can be done through the Management Console as well as the AWS CLI, but for the sake of convenience, we will only cover doing it though former.
```